src/components/SkeletonBodyText/SkeletonBodyText.vue

```diff
@@ -0,0 +1,26 @@
+<template lang="pug">
+div(
+  v-for="index in props.lines" :key="index",
+  :class="styles.SkeletonBodyText",
+)
+</template>
+
+<script setup lang="ts">
+import styles from '@/classes/SkeletonBodyText.json'
+
+interface Props {
+  /**
+   * Number of lines to display
+   * @default 3
+   */
+  lines?: number;
+}
+
+const props = withDefaults(defineProps<Props>(), {lines: 3});
```

Comment:
No need define `const props`
just `withDefaults(defineProps<Props>(), { lines: 3 })` is enough

Reply:
how to get lines without `props`